<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWordDictionaryUiImpl</class>
 <widget class="QDialog" name="FormWordDictionaryUiImpl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelAvailableDictionaries">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>117</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Dictionaries</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDictionaries">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>33</y>
     <width>248</width>
     <height>186</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUp">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>229</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDown">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>229</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>277</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>277</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
